apps/designer/postcss.config.cjs

```diff
@@ -0,0 +1 @@
+module.exports = require('config/postcss.config');
```

Comment:
Faced a few errors like- 
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/user/VVplus/apps/designer/postcss.config.js from /home/user/VVplus/node_modules/.pnpm/next@12.2.5_qtpcxnaaarbm4ws7ughq6oxfve/node_modules/next/dist/lib/find-config.js not supported.

postcss.config.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.

Instead rename postcss.config.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /home/user/VVplus/apps/designer/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

So, we changed the file type to .cjs instead of .js

Reply:
to import modules dynamically